infoportal-client/src/features/Database/KoboTable/customization/useCustomColumns.tsx

```diff
@@ -65,11 +65,15 @@ export const useCustomColumns = (): DatatableColumn.Props<KoboMappedAnswer>[] =>
       head: m.paidOn,
       type: 'date',
       render: (row: KoboAnswer<{}, KoboBaseTags & KoboTagStatus>) => {
+        const today = new Date()
```

Comment:
Little details because performance is a important topic on IP. Here you are instantiating a new date for each cells of the table. I think the best is to move today outside of the component, at the global level of the file. Also `Date.new()` is quicker :-)